documentation/modules/cruise-control/proc-implementing-rebalance-proposal.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='proc-implementing-optimization-proposal-{context}']
+
+= Implementing an optimization proposal
+
+You can implement an optimization proposal with a status of `ProposalReady`.
+Cruise Control will then reassign partitions to brokers as summarized in the optimization proposal.
+
+WARNING: This is not a dry run. 
+Review the optimization proposal before you implement it.
+
+.Prerequisites
+
+* You have xref:proc-generating-optimization-proposals-{context}[generated an optimization proposal].
+
+* The optimization proposal is ready to implement (the `KafkaRebalance` custom resource status is `ProposalReady`).
+
+.Procedure
+
+Perform these steps for the optimization proposal that you want to implement:
+
+. If the `KafkaRebalance` status has been `ProposalReady` for some time, refresh the optimization proposal in case it has become out of date:
+
+.. Annotate the `KafkaRebalance` resource in Kubernetes:
++
+[source,shell,subs="+quotes"]
+----
+kubectl annotate kafkarebalance _cluster-name_-kafka strimzi.io/rebalance=refresh
```

Comment:
The name of the `KafkaRebalance` resources AFAIU can be anything. So the example commands like this should not use the `_cluster-name_-kafka` as a name. They should use something generic like `_kafka-rebalance-name_` instead. Otherwise this would be misleading given the other namesfor example in example do not follow this pattern anyway. (this is not just here but on more places)

Reply:
Good point. I'll change the variable to `_cluster-rebalance-name_` throughout.